WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -584,6 +585,25 @@ private extension OrderListViewController {
         }
     }
 
+    /// Focus code crashes on iPadOS 16 and 16.1 versions
+    /// peaMlT-Ng-p2
+    /// https://github.com/woocommerce/woocommerce-ios/issues/13485
+    private func supportsFocus() -> Bool {
+        guard UIDevice.current.userInterfaceIdiom == .pad else {
+            return true
+        }
+
+        let systemVersion = ProcessInfo.processInfo.operatingSystemVersion
+        let majorVersion = systemVersion.majorVersion
+        let minorVersion = systemVersion.minorVersion
+
+        if majorVersion == 16 && (minorVersion == 0 || minorVersion == 1) {
+            return false
+        }
+
+        return true
```

Comment:
```suggestion
        let iOS16dot2 = OperatingSystemVersion(majorVersion: 16, minorVersion: 2, patchVersion: 0)

        return ProcessInfo().isOperatingSystemAtLeast(iOS16dot2)
```

This is equivalent, and I've checked it – just seems a little more readable. Up to you though, either approach is fine.

Reply:
Alternatively, this might be easier to spot with a search when we drop iOS 16?

```suggestion
        if #available(iOS 16.2, *) {
            return true
        } else {
            return false
        }
```

Reply:
> Alternatively, this might be easier to spot with a search when we drop iOS 16?

Yes, that's a great idea!